testfiles/LookupTables.idf

```diff
@@ -2671,7 +2671,7 @@
 
   Table:OneIndependentVariable,
     FanPowerRatioCurve,      !- Name
-    Exponent,                !- Curve Type
+    ,                        !- Curve Type
```

Comment:
For some reason LookupTables.idf is showing [diffs](http://energyplus.s3-website-us-east-1.amazonaws.com/regressions/2018-08/ce4a92e6e6e86bceade8cfa7ea9ed7b22b9072f2-fb712c7ebe128ce78e7b0b7636645d64f50e1416/LookupTables/x86_64-MacOS-10.9-clang/) in results.  Is it because of this change? Shouldn't matter with LagrangeinterpolationLinearExtrapolation but maybe it does?

Reply:
I'm looking into it. I don't thin it's related to the change in the IDF...I'm thinking there's something else going on with changes in the code.

Reply:
So it turns out "Exponent" isn't valid for regression, but it was used to set the Lagrange interpolation order (to 4). I had been under the impression that Lagrangian interpolation used all of the points in the independent variable to fit a corresponding degree polynomial, but the default is actually zero (set by the constructor). This was never encountered before because the curve type was required.

I'm thinking we just use a default of 4 (i.e., cubic interpolation) if a curve type is not given. That should clear up this diff and give a reasonable interpolation for undefined curve types. I'll change the documentation accordingly.